src/api/app/models/user.rb

```diff
@@ -74,9 +74,14 @@ class User < ApplicationRecord
   validates :login,
             uniqueness: { message: 'is the name of an already existing user' }
 
+  # returns the login validation patern and the errors message
+  def self.login_pattern
+    ['[-+a-zA-Z0-9][-+\w\.]*', 'only allows letters, numbers and the following characters: _ - + t . (. and _ not at the beginning)']
```

Comment:
only allows letters, numbers and the following characters: _ - + **t** .

Hum... what is **t** doing there?

Reply:
I don't know... :confounded: